addons/event/static/src/template_reference_field/field_event_mail_template_reference.js

```diff
@@ -0,0 +1,36 @@
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { ReferenceField, referenceField } from "@web/views/fields/reference/reference_field";
+
+export class EventMailTemplateReferenceField extends ReferenceField {
+    static template = "event.mailTemplateReferenceField";
+
+    setup() {
+        const returnVal = super.setup();
+        const defaultSelect = this.selection?.[0][0];
+        if (defaultSelect) {
+            this.state.currentRelation = defaultSelect;
+        }
```

Comment:
Why do we need this?
Is the "currentRelation" state not set when the component is in readonly?

`this.selection?.[0][0]` <- maybe add an example of the structure inside this selection? Chained 0 index access is a bit meh but I guess there is no way to get this info otherwise.